ras_party/controllers/account_controller.py

```diff
@@ -230,13 +233,14 @@ def change_respondent_password(token, payload, tran, session):
         duration = current_app.config["EMAIL_TOKEN_EXPIRY"]
         email_address = decode_email_token(token, duration)
     except SignatureExpired:
-        raise RasError('Expired email verification token', status=409, token=token)
+        raise ClientError('Expired email verification token', status=409, token=token)
     except (BadSignature, BadData) as e:
         raise RasError('Unknown email verification token', status=404, token=token, error=e)
 
     respondent = query_respondent_by_email(email_address, session)
     if not respondent:
-        raise RasError("Respondent does not exist.", status=404)
+        raise RasError("Respondent does not exist. Unable to change respondent's password",
```

Comment:
@gavinleeedwards Do we want info level here?

Reply:
ignore, just seen the `ClientError`

Reply:
I think we need to be careful about trying to be too helpful in the log messages and repeating information already in the log message. For example here, the context of the respondent not existing should be clear if we're logging the url, without needing to add "Unable to change respondent's password"

Reply:
I don't necessarily agree that being too helpful is a bad thing.  The issue here is that `Respondent does not exist` could come from several places in the code so making sort of messages less generic makes it quicker to pinpoint where any issues might be occurring.

Reply:
I think we'll end up with a lot of duplication. 'Expired email verification token' and 'Unknown email verification token' appear multiple times but haven't had any more logging added. If we applied the same 'Unable to change respondent's password' to all of those log lines there would be a lot of repetition. If the url was being logged we'd see 'Respondent does not exist' along with `url=/respondents/change_password/<token>, making it clear what action was being performed when the error occurred.